### PR TITLE
Update Readme: encrypt returns a string not an object

### PR DESCRIPTION
**Pull request by radko93**
Readme file is misleading - encrypt returns `Promise<string>` and not an object.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +93,7 @@ try {
     generateKey('Arnold', 'salt', 5000, 256).then(key => {
         console.log('Key:', key)
         encrypt('These violent delights have violent ends', key)
-            .then(({ cipher, iv }) => {
+            .then((cipher) => {
                 console.log('Encrypted:', cipher)
 
                 decrypt({ cipher, iv }, key)
```
Comment (radko93) [inline README.md:99]: This has to be updated probably as well.

Comment (radko93) [inline README.md:99]: This has to be updated probably as well as `iv` will be missing

**Comment by mobitar** [comment]
If that's the case then

```
return Aes.encrypt(text, key, iv).then(cipher => {
   return cipher;
})
```

can just be simplified to 

```
return Aes.encrypt(text, key, iv)
```

**Comment by tectiv3** [comment]
hm, I don't think it's misleading. It does return Promise:
```js
...
      encrypt('These violent delights have violent ends', key)
        .then(({cipher, iv}) => {
          this.setState({cipher, iv, key});
...
```
```
                <Text style={styles.sectionDescription}>
                  cipher: {this.state.cipher}
                </Text>
                <Text style={styles.sectionDescription}>
                  iv: {this.state.iv}
                </Text>
```
Otherwise you would lose `iv`.
Screenshot 2020-04-29 at 21 24 50

**Comment by tectiv3** [comment]
I don't think it is misleading. Yes `Aes.encrypt` returns `Promise<string>` but `encrypt` from the readme example does return object - `cipher` and `iv`.

**Comment by radko93** [comment]
@tectiv3 I updated naming so the function that does not use the same name. Still, Aes.encrypt in `encryptData` was returning a wrong value

**Pull request merged**